movies/src/actions/index.js

```diff
@@ -0,0 +1,20 @@
+import axios from 'axios';
+
+export const GET_MOVIES = 'GET_MOVIES';
+export const PICK_MOVIE = 'PICK_MOVIE';
+
+export const getMovies = () => {
+    const promise = axios.get('http://localhost:5000/movies');
+    return ({
```

Comment:
You can get rid of the outer parentheses here around the action object you're returning.